### ISSUE TITLE
Question: is there a way for blocking external links?

### ISSUE THREAD

**Issue by fernandoabolafio** [rexxars/react-markdown#124]

**Comment by rexxars** [comment]
Yes. `transformLinkUri` allows you to rewrite links. You could also define a custom renderer for the `link` type, which would be the most flexible solution. This would allow you to render an error message, rewrite the link, just render the children of the link or any other thing you might want to do.

**Comment by fernandoabolafio** [comment]
Awesome. Thanks for the quick answer.
